engine/schema/mysql.sql

```diff
@@ -147,36 +127,23 @@ CREATE TABLE `prefix_metadata` (
   `entity_guid` bigint(20) unsigned NOT NULL,
   `name_id` int(11) NOT NULL,
   `value_id` int(11) NOT NULL,
-  `value_type` enum('integer','text') NOT NULL,
-  `owner_guid` bigint(20) unsigned NOT NULL,
```

Comment:
Access control on metadata causes more problems than solves IMO. If you need it, use an entity or annotation